src/bpftrace.cpp

```diff
@@ -693,17 +693,17 @@ int BPFtrace::run(Output &out, BpfBytecode bytecode)
   int num_begin_end_attached = 0;
   int num_signal_attached = 0;
 
-  auto begin_probe = resources.special_probes.find("BEGIN");
+  auto begin_probe = resources.special_probes.find("begin");
   if (begin_probe != resources.special_probes.end()) {
     auto &begin_prog = bytecode_.getProgramForProbe((*begin_probe).second);
     if (::bpf_prog_test_run_opts(begin_prog.fd(), nullptr))
       return -1;
 
-    LOG(V1) << "Attaching BEGIN";
+    LOG(V1) << "Attaching begin";
```

Comment:
Nit: here and below it's not clear if this is the probe name or just bad grammar. Can we quote the probe? E.g. "Attaching `begin` probe"

Reply:
Done.